sources/packages/backend/apps/api/src/services/student-file/student-file.service.ts

```diff
@@ -246,29 +276,31 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
   }
 
   /**
-   * Gets a list of student files from the student account
-   * (i.e, fileOrigin is FileOriginType.Student or FileOriginType.Ministry).
-   * @param studentId student id.
-   * @returns student files from the student account.
+   * Uploads the file to the S3 file storage.
+   * @params file the file to be uploaded to the file storage.
    */
-  async getStudentUploadedFiles(studentId: number): Promise<StudentFile[]> {
-    return this.repo
-      .createQueryBuilder("studentFile")
-      .select([
-        "studentFile.uniqueFileName",
-        "studentFile.fileName",
-        "studentFile.metadata",
-        "studentFile.groupName",
-        "studentFile.updatedAt",
-        "studentFile.fileOrigin",
-      ])
-      .where("studentFile.student.id = :studentId", { studentId })
-      .andWhere("studentFile.fileOrigin IN (:...fileOrigin)", {
-        fileOrigin: [FileOriginType.Student, FileOriginType.Ministry],
-      })
-      .orderBy("studentFile.createdAt", "DESC")
-      .getMany();
+  private async uploadFileToStorage(file: CreateFile): Promise<void> {
+    try {
+      this.logger.log(`Uploading file ${file.fileName} to S3 storage.`);
+      await this.objectStorageService.putObject({
+        key: file.uniqueFileName,
+        contentType: file.mimeType,
+        body: file.fileContent,
+      });
```

Comment:
I am not sure if I see a benefit of this method being private anymore. Can we just move the method call to the 285 inside the other method?